README.md

```diff
@@ -23,13 +23,22 @@ content-cli profile create
 **Pull:** This feature allows you to download content from the EMS to your local machine. Let's take Studio package as an example. These can be exported in the EMS as ZIP files that contain all package assets. By using the following command using the package key and profile you have created, you will pull the ZIP file.
 
 ```
-content-cli pull package -p team1.cluster1 --key my-package
+content-cli pull package -p {{profile-name}} --key {{package-key}}
+```
+Example, if you want to pull a package with key "interesting-package" from your profile "my-dev-profile", the command will look like this:
+```
+content-cli pull package -p interesting-package --key my-dev-profile
 ```
 
 **Push:** This feature allows you to push a content file to a team in the EMS. To continue the last example, you can use the following command to push he previously pulled package in another team.
 
 ```
-content-cli push package -p team2.cluster2 --spaceKey my-space -f package_my-package.zip
+content-cli push package -p {{profile-name}} --spaceKey {{space-key}} -f {{file-to-push.zip}}
```

Comment:
Can we use `< >` notations because they also appear when running the `--help` flag with a command. 
Also there a few more places in [documentation](https://github.com/celonis/content-cli/blob/master/DOCUMENTATION.md) that would need to be wrapped like this to have them consistent.

Reply:
Yes nice, will switch to that.